test/format/ops.c

```diff
@@ -1027,6 +1054,16 @@ ops(void *arg)
 		 */
 		switch (rnd) {
 		case 1: case 2: case 3: case 4:			/* 40% */
+			/*
+			 * Occasionally prepare the transaction too.
+			 */
+			if (rnd == 1) {
+				ret = prepare_transaction(tinfo, session);
+				testutil_assert(
+				    ret == 0 || ret == WT_PREPARE_CONFLICT);
+				if (ret == WT_PREPARE_CONFLICT)
+					goto deadlock;
+			}
```

Comment:
I'd suggest:
```
/*
 * If we're in a transaction, commit 40% of the time and
 * rollback 10% of the time.
 */
switch (rnd) {
case 1:                                         /* 10% */
        /*
         * Occasionally prepare the transaction, then commit.
         */
        ret = prepare_transaction(tinfo, session);
        testutil_assert(ret == 0 || ret == WT_PREPARE_CONFLICT);
        if (ret == WT_PREPARE_CONFLICT)
                goto deadlock;
        /* FALLTHROUGH */
case 2: case 3: case 4:                         /* 40% */
        commit_transaction(tinfo, session);
        break;
case 5:                                         /* 10% */
```
rather than testing `rnd` explicitly, but I don't feel strongly.

Reply:
Fixed.